Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -92,7 +92,8 @@ bash scripts/install_prerequisites.sh
 ## Setup a virtual environment
 
 For your virtual environment, we recommend you use [pyenv](https://github.com/pyenv/pyenv). Here are some instructions for doing so, but you can use another virtual environment tool if you wish:
-
+   > [NOTE!]
+   > Below commands can be executed in any directory, as they are not path-specific. However, new contributors should execute in the folder (``cd opensource/``) to prevent any confusion.
```

Comment:
Below commands --> The commands below

Also, I don't think you need the "However" statement. It really doesn't matter where you execute these since "apt-get install" isn't specific to a particular directory.

Reply:
Done!